### ISSUE TITLE
Fail to disassemble RISCV code

### ISSUE THREAD

**Issue by itewqq** [NationalSecurityAgency/ghidra#5932]
**Describe the bug**
I'm reversing a firmware extracted from a RISCV chip, and there are some instructions that ghidra cannot handle properly, like the bytes `8b 15 a5 00` at 0xa0946 in the following figure:
![image](https://github.com/NationalSecurityAgency/ghidra/assets/30570177/ffa20232-4a32-4e06-931f-eb03421663ef)

**To Reproduce**
1. Select `RV32GC` in as the language.
2. Just try to disassemble the above PoC bytes.

**Expected behavior**
Disassembly or at least the reason why it fails for these bytes?

**Screenshots**
See above.

**Attachments**
If applicable, please attach any files that caused problems or log files generated by the software.

**Environment (please complete the following information):**
 - OS: Archlinux, latest (2023-11-06).
 - Java Version: 11.0
 - Ghidra Version: 10.4
 - Ghidra Origin: official GitHub distro

**Comment by mumbel** [comment]
I'd have to think more on why/how I added the generic custom instructions to the sleigh, but looks like whatever i used to generate that file missed op1214=1 and op1214=5

this is a custom instruction.  It would need a special implementation to be meaningful even if it did disassemble.

There are some issues/pull requests looking at adding `thead`, but your post is light on details so not sure if those would be helpful.

to get it disassembling you can change `<>/Ghidra/Processors/RISCV/data/languages/riscv.custom.sinc`
```
:custom0 is op0001=0x3 & op0204=0x2 & op0506=0x0 & op1214=0x0                                        
{                                                                                                    
        custom0();                                                                                   
}                                                                                                    
 ```
to
```
:custom0 is op0001=0x3 & op0204=0x2 & op0506=0x0 & (op1214=0x0 | op1214=0x1 | op1214=0x5)                     
{                                                                                                    
        custom0();                                                                                   
}                                                                                                    
```

**Comment by GhidorahRex** [comment]
@mumbel It looks like these should possibly be added for the other custom 1/2/3 instructions as well, yes?

**Comment by mumbel** [comment]
Correct, I should have said that. The custom0 was just this specific decoding
